pkg/arvo/app/contact-view.hoon

```diff
@@ -181,22 +181,7 @@
     ~[(add-pending rid ship.act)]
   ::
       %delete
-    =/  rid=resource
-      (de-path:resource path.act)
-    =/  group-pokes=(list card)
-      ?:  =(our.bol entity.rid)
-        ~[(group-push-poke %remove rid)]
-      :~  (group-proxy-poke %remove-members rid (sy our.bol ~))
-          (group-pull-poke %remove rid)
-      ==
-    ;:  weld
-      group-pokes
-      :~  (contact-hook-poke [%remove path.act])
-          (group-poke [%remove-group rid ~])
-          (contact-poke [%delete path.act])
-      ==
-      (delete-metadata path.act)
-    ==
```

Comment:
@ixv @liam-fitzgerald Does this impact #3604? I believe we wanted a freeze on all code touching that PR as per @philipcmonk. Should we revert this PR back out of next-userspace if so?

Reply:
We could remove the modification to %contact-view without harming anything.

Reply:
It should be fine, but I'll let @ixv confirm

Reply:
I don't believe so, but I'm about to do another round of testing so I will try it out.

Reply:
just finished testing, its all good